Sources/AblyAssetTrackingSubscriber/Exports.swift

```diff
@@ -0,0 +1,3 @@
+// This 
+// There have been proposals to work e.g. `public import AblyAssetTrackingCore` to expose the import to users of this module
+@_exported import AblyAssetTrackingCore
```

Comment:
https://forums.swift.org/t/what-does-exported-import-do/35869/4

Reply:
Yea, so it is not stable. In the future, in a future version of Swift, it will probably be removed and replaced with `public`, at which point that file should be updated. This is to avoid doing `import AblyAssetTrackingCore` on top of `import AblyAssetTrackingPublisher`. 

Do you think I should not have it at all? LMK. I don't think it's dangerous to keep it.

Reply:
well, AAT is still in Beta so it's safe for now, but even if `public` will be introduced in a future version of swift, this will require updating Xcode IDE from users. Leave it for now, keep it in mind for future 😅